native/core/src/models.rs

```diff
@@ -270,6 +278,9 @@ pub enum AnyActerModel {
     AttachmentUpdate(AttachmentUpdate),
 
     Rsvp(Rsvp),
+    Reaction(Reaction),
+
+    Redaction(Redaction),
```

Comment:
this doesn't make sense to me. See below. a "Redaction" isn't meant to be a special type of event ...

Reply:
Fixed by https://github.com/acterglobal/a3/pull/1208/commits/d50b2841f52227a6a85d51b8dc2d0fe16464a7f4